logstash-output-google_cloud_storage.gemspec

```diff
@@ -22,10 +22,14 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
 
   s.add_runtime_dependency 'stud'
-  s.add_runtime_dependency 'google-api-client', '~> 0.8.7' # version 0.9.x works only with ruby 2.x
   s.add_runtime_dependency 'logstash-codec-plain'
   s.add_runtime_dependency 'mime-types', '~> 2' # last version compatible with ruby 2.x
   s.add_runtime_dependency 'concurrent-ruby', '1.0.5'
   s.add_development_dependency 'logstash-devutils'
+
+  # JARs
+  s.add_development_dependency 'jar-dependencies', '~> 0.3.4'
```

Comment:
this should not be required, in fact, locally I had to remove it to be able to run spec because of a version conflict.